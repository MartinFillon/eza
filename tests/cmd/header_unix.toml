bin.name = "eza"
args = "tests/itest --header --long"
