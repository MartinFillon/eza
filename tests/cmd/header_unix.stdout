Permissions Size User Date Modified  Name
.rw-rw-r--     0 mfn  12 sept. 10:02 a
.rw-rw-r--     0 mfn  12 sept. 10:02 b
.rw-rw-r--     0 mfn  12 sept. 10:02 c
.rw-rw-r--     0 mfn  12 sept. 10:02 d
.rw-rw-r--     0 mfn  12 sept. 10:02 e
.rw-rw-r--     0 mfn  12 sept. 10:02 f
.rw-rw-r--     0 mfn  12 sept. 10:02 g
.rw-rw-r--     0 mfn  12 sept. 10:02 h
.rw-rw-r--     0 mfn  12 sept. 10:02 i
.rw-rw-r--     0 mfn  12 sept. 10:02 j
.rw-rw-r--     0 mfn  12 sept. 10:02 k
.rw-rw-r--     0 mfn  12 sept. 10:02 l
.rw-rw-r--     0 mfn  12 sept. 10:02 m
.rw-rw-r--     0 mfn  12 sept. 10:02 n
.rw-rw-r--     0 mfn  12 sept. 10:02 o
.rw-rw-r--     0 mfn  12 sept. 10:02 p
.rw-rw-r--     0 mfn  12 sept. 10:02 q
